### PR TITLE
Doc: update course video data response values

### PR DESCRIPTION
**Pull request by mhoeber**
@dsjen @lamagnifica

### CODE REVIEW

**Comment by dsjen** [comment]
Looks great! :+1:

File: docs/api/source/change_log.rst [hunk 1]
```diff
@@ -8,5 +8,8 @@ Change Log
 
    * - Date
      - Change
-   * - 05/18/15
+   * - 21 May 2015
+     - Updated :ref:`Get the Course Video Data` to add the ``users_at_start``
```
Comment (lamagnifica) [inline docs/api/source/change_log.rst]: add or rename? looks like a name change maybe?

**Comment by lamagnifica** [comment]
1 possible clarification in the change log. :+1:

File: docs/api/source/change_log.rst (hunk 1 above)
Comment (mhoeber) [inline docs/api/source/change_log.rst]: it's not really a rename - it's different fields

**Pull request merged**